Makefile

```diff
@@ -351,9 +351,12 @@ else ifneq (,$(findstring RK,$(platform)))
 else ifneq (,$(findstring osx,$(platform)))
    TARGET := $(TARGET_NAME)_libretro.dylib
    LDFLAGS += -dynamiclib
-   OSXVER = `sw_vers -productVersion | cut -d. -f 2`
+
+   OSXVER = $(shell sw_vers -productVersion | cut -d. -f 2)
    OSX_LT_MAVERICKS = `(( $(OSXVER) <= 9)) && echo "YES"`
-        LDFLAGS += -mmacosx-version-min=10.7
+   ifeq ($(OSX_LT_MAVERICKS),YES)
```

Comment:
can we just ifeq(,1)?

Reply:
This one was copied from the makefile of snes9x, which is knows to work. Without it, we get this error at link time:
```
clang: error: invalid deployment target for -stdlib=libc++ (requires OS X 10.7 or later)
```